terminus-ui/popover/src/popover.component.html

```diff
@@ -0,0 +1,17 @@
+<div
+  #popoverViewRef
+  class="popover__container"
+  [ngClass]="{
+    'c-popover__visible': visibility,
+    'c-popover__hidden': !visibility
```

Comment:
These classes are using the 'element' style rather than the 'modifier' style

eg

```
.car {}
.car__door {}
.car--blue {}
```